### DISCUSSION TITLE
Shortcut for opening the application

### DISCUSSION THREAD

**Discussion by tomikonio** [keepassxreboot/keepassxc#9236] [Q&A]
Hi,
I am using keepass2 and while it's running and minimized, I can bring it to the screen with `CTRL+ALT+K`.
Is there a way to do this with keepassxc?- I did not find anything in the settings.

**Comment by droidmonkey** [comment]
It's a requested feature

**Reply by tomikonio** [reply]
Thanks, @droidmonkey.
Is there an open issue on the subject?

**Reply by droidmonkey** [reply]
There is, #99
